Yoast/YoastSEO.js
# Change JSON files to JS

In order to not have to copy json files when creating a publish build it's easier to have JS files instead of JSON files because they will be picked up by babel.

## Test instructions
* Run `grunt publish`
* Inspect `dist` folder and see if the language files are there (de.js, it.js, en.js and nl.js).
